kube/services/ssjdispatcher/rolebinding-devops.yaml

```diff
@@ -0,0 +1,12 @@
+kind: RoleBinding
```

Comment:
rename this file to `ssjdispatcher-binding.yaml`

Reply:
Rename in a55945ffa92f01cdd9dc040a8fa35656ee373220